graphix/clifford.py

```diff
@@ -219,3 +219,30 @@
     ["h", "x", "sdg"],
     ["h", "x", "s"],
 ]
+
+CLIFFORD_TO_QISKIT = [
```

Comment:
what's the difference from CLIFFORD_TO_QASM3?

Reply:
Initially, I thought the identity operation in qiskit would be written as "I", so I created a new list, but eventually I found out that it is also "id" in qiskit, so the two became identical.
I think we can delete it.

Reply:
@d1ssk shall we delete this CLIFFORD_TO_QISKIT and use CLIFFORD_TO_QASM3, if they're identical?

Reply:
I deleted it.